executor/show_test.go

```diff
@@ -310,12 +310,9 @@ func (s *testSuite) TestShow(c *C) {
 	))
 
 	tk.MustExec(`drop table if exists t`)
-	tk.MustExec(`create table t (a int) default charset=abcdefg`)
```

Comment:
We don't need to remove the test case, we can change the charset to a valid one like `utf8mb4`.

Reply:
Already included the correct test cases in https://github.com/pingcap/tidb/blob/master/executor/show_test.go#L305